src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -117,7 +124,9 @@ public function update( $new_instance, $old_instance ) {
 		$updated_instance['jsonld_enable']        = (int) ( ! empty( $new_instance['jsonld_enable'] ) );
 		$updated_instance['tribe_is_list_widget'] = ! empty( $new_instance['tribe_is_list_widget'] );
 
-		return $this->filter_updated_instance( $updated_instance, $new_instance );
```

Comment:
Any specific reason for this outside of debugging in-line? Please restore it and get some micro-optimization going here.